canonical-websites/www.ubuntu.com
# V1 cloud partners

## Done

/cloud/partners to vbt

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/cloud/partners](http://0.0.0.0:8001/cloud/partners))
- Check the page against the design https://www.dropbox.com/home/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/2.%20Cloud?preview=cloud-partners.png


## Issue / Card

Fixes #1614 

Design: https://www.dropbox.com/home/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/2.%20Cloud?preview=cloud-partners.png
